app/shared/pathsToAllowThroughWithoutSignin.ts

```diff
@@ -0,0 +1,22 @@
+export const pathsToAllowThroughWithoutSignin = [
+  "/contact-us-form",
+  "/contact-us-form/*",
+  "/contact-us-form/*/*",
+  "/contact-us-form/*/*/*"
+];
+
+export const allowPathThroughWithoutSignin = (path: string) => {
```

Comment:
This seems critical? How do you know it works? Could we write some tests?

Reply:
we have written some tests :)